ui/views_test.py

```diff
@@ -37,3 +40,30 @@ def test_upload_admin(admin_client, mocker):
     assert response.status_code == 200
     assert response.context_data['dropbox_key'] == 'dbkey'
     assert response.template_name == ['ui/upload.html']
+
+
+def test_video_detail_hls(admin_client, video, videofileHLS, mocker):  # pylint: disable=unused-argument
+    """Test video detail page when HLS videofile is available"""
+    mocker.patch('ui.models.make_cloudfront_signed_url', return_value=videofileHLS.cloudfront_url)
+    video.status = 'Complete'
+    url = '/videos/{}/'.format(video.id)
+    response = admin_client.get(url)
+    assert 'videofile' in response.context_data
+    assert response.context_data is not None
```

Comment:
This is already checked due to the previous assertion. Also, I'm not sure the check is necessary since the check below will fail if it's not present.